### ISSUE TITLE
Update the latest tag on npm to match the latest version 

### ISSUE THREAD

**Issue by dsifford** [typicode/husky#434]
Not a huge problem by any means, but it's sort of annoying that `npm outdated` constantly reports this as being outdated when using `v1.3.0` (assuming `v1.3.0` is correctly labelled as not a pre-release version).

```sh
$ npm outdated
Package           Current  Wanted  Latest  Location
husky               1.3.0   1.3.0   1.2.1  project

$ npm view husky dist-tags
{ latest: '1.2.1' }
```

**Comment by jaydenseric** [comment]
It looks like this is deliberate, see https://github.com/typicode/husky/issues/431#issuecomment-449400908.

**Comment by typicode** [comment]
> It looks like this is deliberate, see #431 (comment).

Yes and temporary :)

**Comment by typicode** [comment]
Just published `1.3.1`, should be good now :+1:
